tchannel/peer_heap.py

```diff
@@ -0,0 +1,61 @@
+from __future__ import absolute_import
+
+from .container import heap
+from .container.heap import HeapOperation
+
+
+class PeerHeap(HeapOperation):
+    def __init__(self):
+        self.peers = []
+
+    def cmp(self, i, j):
+        return self.peers[i] < self.peers[j]
```

Comment:
Oh that's because this is just the simplest version. The real version won't be just compare score only.  Those changes will appear in the future diffs .